core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.ts

```diff
@@ -147,6 +149,9 @@ export class AudioBarComponent {
   expandAudioBar(): void {
     this.audioBarIsExpanded = true;
     this.audioBarStatusService.markAudioBarExpanded();
+    setTimeout(() => {
+      this.audioBarContainer.nativeElement.focus();
+    });
```

Comment:
I guess you need to add spec for these lines, i think frontend coverage is failing because of this